pkg/volume/volume.go

```diff
@@ -45,6 +45,12 @@ type Builder interface {
 	// IsReadOnly is a flag that gives the builder's ReadOnly attribute.
 	// All persistent volumes have a private readOnly flag in their builders.
 	IsReadOnly() bool
+	// RequiresSELinuxRelabel reports whether the given builder
+	// supports SELinux and would like the kubelet to relabel the
+	// volume to match the pod to which it will be attached.
+	// Usually this returns true of the volume type supports
+	// SELinux and the volume is newly formatted.
+	RequiresSELinuxRelabel() bool
```

Comment:
This is called `SupportsSELinux` in the proposal, I'd prefer to stick with the names we agreed to (even though I don't think they're great) and we can bikeshed around this once the functionality is working.  There's a lot of cleanup to be done on volumes after this, and we'll make it just-so then.